include/astshim/FrameSet.h

```diff
@@ -100,8 +100,8 @@ class FrameSet : public Frame {
 friend class Frame;  // so Frame can call the protected raw ptr constructor
 friend class Object;
 public:
-    static int const BASE;  ///< index of base frame
-    static int const CURRENT;  ///< index of current frame
+    static int constexpr BASE = AST__BASE;  ///< index of base frame
+    static int constexpr CURRENT = AST__CURRENT;  ///< index of current frame
```

Comment:
Note that without making, e.g., `getFrame` and `Frame` support constexpr, this change is unlikely to be useful. But that's clearly out of scope for this ticket (and `getFrame`, at least, is too complex to make `constexpr` before C++14).